src/goInstallTools.ts

```diff
@@ -40,6 +42,8 @@ const _allTools: { [key: string]: string } = {
 	'golangci-lint': 'github.com/golangci/golangci-lint/cmd/golangci-lint',
 	'revive': 'github.com/mgechev/revive',
 	'go-langserver': 'github.com/sourcegraph/go-langserver',
+	'bingo': 'github.com/saibing/bingo',
+	'golsp': 'golang.org/x/tools/cmd/golsp',
```

Comment:
@stamblerre Would this be the right import path to use in `go get` for the language server from your team?